packages/frontend/pages/provenance/[deviceKey].vue

```diff
@@ -13,7 +13,9 @@ const deviceKey = route.params.deviceKey;
     <div>
     <ProvenancePriorityNotices :deviceKey="deviceKey" :provenance="provenance"/>
     </div>
-    <a href="#createRecord">Go to "Create New Provenance Record"</a>
+    <a href = "#createdDevicePoint"><button class = "textToLinkButton0">Click to <i><textToLink class = "textToLink">Device Creation</textToLink></i></button></a>
```

Comment:
@noramoor I can't figure out where the "textToLink" formal class is defined in your branch. This is thowing an error (albeit a benign one) in my branch. Did you perhaps forget to check in a file?  Based on the CSS below, did you mean for this to be <span class="textToLin">?